pkg/grpc/server.go

```diff
@@ -117,24 +116,28 @@ func NewServersFromConfigurationAndServe(configurations []*configuration.ServerC
 
 		// TCP sockets.
 		for _, listenAddress := range configuration.ListenAddresses {
+			// Make a copy for the go routine.
+			listenAddress := listenAddress
```

Comment:
Nit: I'm generally not a fan of shadowing of existing variables. Could you change the loop itself to use `listenAddressIter`? No need to add the comment above; It's sort of unwritten 'common practice' in the Buildbarn tree to use `Iter` as the suffix.

Reply:
Done that. Waiting for the GitHub actions to run.